### PR TITLE
APMSPII-216 | Update Telecom

### PR DESCRIPTION
**Pull request by ben-clarke**
## Summary
* Add update telecom description

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/personal-demographics.yaml [hunk 1]
```diff
@@ -1275,6 +1380,20 @@ paths:
...
+        List item found under `telecom` field.
+
+        In a JSON Patch request the path should be:
+        * `/telecom/0` if the telecom to be replaced or removed is the first item in the list. If it is the second item in the list the path will be `/telecom/1`, and so on.
+        * `/telecom/-` when adding a new telecom.
+
+        Emergency contact details should not be added to the `telecom` field, instead they should be added to the `contact` field.
+
+        There are a number of business rules that should be taken into account:
+        * the date period is optional; where present they must a be valid dates and the `end` date cannot be before the `start` date.
```
Comment (alisonkinloch-nhs) [inline specification/personal-demographics.yaml:1394]: typo on a

File: specification/personal-demographics.yaml [hunk 2]
```diff
@@ -1082,7 +1082,44 @@ paths:
         *	Line 4 - post town, e.g. `Leeds`
         *	Line 5 - county (if present), e.g. `West Yorkshire`.
 
-        When updating an address, lines 1 or 2, and line 4 must be populated; line 5 should not be included in vernacular addresses but may be included in PAF-derived addresses.
+        When updating an address, lines 1 or 2, and line 4 must be populated; line 5 should not be included in manually created addresses but may be included in PAF derived addresses.
+
+        Address lines should be populated otherwise the message may be rejected. There are exceptions:
+        * if a postcode and PAF key are submitted, in which case the lines will be automatically populated, however if there are no matches or too many matches they message will be rejected due to missing address lines.
+        * a pseudo postcode is used. For example `ZZ99 3VZ` meaning `no fixed abode`.
```
Comment (alisonkinloch-nhs) [inline specification/personal-demographics.yaml:1089]: can a link be put in here to the pseudo postcode list

Comment (alisonkinloch-nhs) [reply specification/personal-demographics.yaml:1089]: Typo on "they"

File: specification/personal-demographics.yaml (hunk 1 above)
Comment (ben-clarke) [reply specification/personal-demographics.yaml:1394]: Fixed

File: specification/personal-demographics.yaml (hunk 2 above)
Comment (ben-clarke) [reply specification/personal-demographics.yaml:1089]: Fixed typo - linking within an apigee doc apparently does not work, so cannot add a link to the same page.

**Pull request closed without merging**